src/web/assets/cp/src/css/_cp.scss

```diff
@@ -1485,6 +1485,20 @@ li.breadcrumb-toggle-wrapper {
   }
 }
 
+.errors-summary {
+  margin-bottom: var(--s);
+  padding: var(--s) var(--xl);
+  box-sizing: border-box;
+  box-shadow: inset 0 -1px 0 0 rgba(154, 165, 177, 0.25);
```

Comment:
```suggestion
  box-shadow: 0 0 0 1px #cdd8e4,0 2px 12px rgba(205,216,228,.5);
```

Reply:
I think matching the box-shadow being used by the `.content-pane` and `.meta` containers will help make this container more cohesive. Perhaps create a new CSS variable for this box-shadow?